react/features/notifications/components/native/Notification.tsx

```diff
@@ -0,0 +1,302 @@
+        const { _participants, icon } = this.props;
+        const oneParticipant = _participants.length === 1;
+        const contentColumnStyles = icon === NOTIFICATION_ICON.PARTICIPANTS
+            ? styles.contentColumn : styles.interactiveContentColumn;
+
+        return (
+            <View
+                pointerEvents = 'box-none'
+                style = { styles.notification }>
+                <View style = { contentColumnStyles }>
+                    { oneParticipant ? this._renderAvatar() : this._mapAppearanceToIcon() }
+                    <View pointerEvents = 'box-none'>
+                        { this._renderContent() }
+                    </View>
+                    <View style = { styles.btnContainer }>
+                        { this._mapAppearanceToButtons() }
+                    </View>
+                </View>
+                <IconButton
+                    color = { BaseTheme.palette.icon04 }
+                    onPress = { this._onDismissed }
+                    src = { IconCloseLarge }
+                    type = { BUTTON_TYPES.TERTIARY } />
+            </View>
+        );
+    }
+
+    /**
+     * Renders the notification's content. If the title or title key is present
+     * it will be just the title. Otherwise it will fallback to description.
+     *
+     * @returns {Array<ReactElement>}
+     * @private
+     */
+    _renderContent() {
+        // @ts-ignore
+        const { icon, maxLines = DEFAULT_MAX_LINES, t, title, titleArguments, titleKey, concatText } = this.props;
+        const titleText = title || (titleKey && t(titleKey, titleArguments));
+        const description = this._getDescription();
+        const titleConcat = [];
+        const descriptionStyles = icon === NOTIFICATION_ICON.PARTICIPANTS
+            ? styles.contentTextInteractive : styles.contentText;
+
+        if (concatText) {
+            titleConcat.push(titleText);
+        }
+
+        if (description?.length) {
+            return [ ...titleConcat, ...description ].map((line, index) => (
+                <Text
+                    key = { index }
+                    numberOfLines = { maxLines }
+                    style = { descriptionStyles }>
+                    { replaceNonUnicodeEmojis(line) }
+                </Text>
+            ));
+        }
+
+        return (
+            <Text
+                numberOfLines = { maxLines }>
+                { titleText }
+            </Text>
+        );
+    }
+
+    /**
+     * Renders the avatar.
+     *
+     * @returns {Array<ReactElement>}
+     * @private
+     */
+    _renderAvatar() {
+        // @ts-ignore
+        const { _participants } = this.props;
+
+        return (
+            <>
+                {
+                    _participants?.map((p: any, index: number) => (
+                        <View
+                            key = { index }
+                            style = { styles.avatarContainer }>
+                            <Avatar
```

Comment:
Why are we displaying avatars here? The web just renders the participant icon, since we don't have a real avatar anyway, and they'd be too small to be meaningful. Please drop this.

Reply:
Latest design shows avatar inside notifications.

Reply:
Let's not. It's so small it's useless.